chall03/npimenof.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	client := http.Client{}
+	res, err := client.Get("https://chall03.hive.fi")
+	if err != nil {
+		log.Fatal(err)
+	}
+	body, err := ioutil.ReadAll(res.Body)
```

Comment:
you should close the res.Body() after using it. You can use defer or just write:

resp.Body().Close() after the error check.